api/openapi/v1/system-agent.yaml

```diff
@@ -32,27 +34,56 @@ paths:
   /v1/metrics/{services}:
     get:
       description: Fetch the operating performance metrics of the specified EdgeX
-        services by their unique names - returning null if none is found for the service.
-        HTTP 500 for unknown or unanticipated issues
+        unique names - returning an error message if a problem is encountered for a
+        given service. And returning HTTP 500 for unknown or unanticipated issues
       parameters:
       - name: services
         in: path
-        description: EdgeX micro service names (the unique name of each service comma
-          delimited)
+        description: EdgeX micro service names (the unique name of each service,
+            comma-delimited)
         required: true
         style: simple
         explode: false
         schema:
           type: string
       responses:
         200:
-          description: an array of metrics
+          description: an array of elements where each element contains the kind
+            of executor that was used to fetch the metrics, the operating
+            performance metrics themselves, and the service name for which
+            the metrics were requested
           content:
             '*/*':
               schema:
                 $ref: '#/components/schemas/metric'
         500:
           description: for unknown or unanticipated issues.
+  /v1/health/{services}:
+    get:
+      description: Fetch the health status of the specified EdgeX services by their
+        unique names - returning either true (for a healthy service) or a given
+        service is not healthy. And returning HTTP 500 for unknown or unanticipated
+        issues
+      parameters:
+        - name: services
+          in: path
+          description: EdgeX micro service names (the unique name of each service,
+            comma-delimited)
```

Comment:
Please avoid the use of parenthesis as described on the `config/{services}` endpoint.